nuget/nunitlite.nuspec

```diff
@@ -37,10 +37,15 @@ How to use this package:
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
     <file src="bin/net-2.0/nunitlite.dll" target="lib/net20" />
+    <file src="bin/net-2.0/nunitlite-runner.exe" target="lib/net20" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
+    <file src="bin/net-4.0/nunitlite-runner.exe" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
+    <file src="bin/net-4.5/nunitlite-runner.exe" target="lib/net45" />
     <file src="bin/portable/nunitlite.dll" target="lib/dotnet" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/dotnet" />
     <file src="bin/portable/nunitlite.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
```

Comment:
Exactly... that's why I put it under portable. :-)

I was planning on writing a blog post about testing portable apps, which includes the two different approaches:
 * Using portable tests
 * Using .NET 4.5 tests
The nunitlite-runner built against the portable library is only intended for the first case, which should be rare except for tests that involve gui elements.

I'll change the directory structure as you suggest. I'll use the standard NuGet names for the directories rather than our names, to avoid confusion.

I'm not giving up entirely on running xamarin tests from the desktop as well! People have said you can't run silverlight or CF tests from the desktop, but I've managed to do both. (CF was in the old NUnitLite project under NAnt. I want to figure out how to do it under Cake rather than wasting time with MsBuild.)